lib/default-options.js

```diff
@@ -16,7 +16,8 @@ module.exports = function defaultOptions () {
             'assert.notDeepEqual(actual, expected, [message])',
             'assert.deepStrictEqual(actual, expected, [message])',
             'assert.notDeepStrictEqual(actual, expected, [message])'
-        ]
+        ],
+        additionalMethods: []
```

Comment:
I am having a much harder time coming up with a good name for this one.

- `wrapOnlyPatterns`
- `wrappedOnlyPatterns`
- `noRenderPatterns`
- `unrenderedPatterns`